collect_app/src/main/java/org/odk/collect/android/views/ODKView.java

```diff
@@ -483,7 +484,22 @@ public void highlightWidget(FormIndex formIndex) {
                 scrollTo(0, qw.getTop());
 
                 ValueAnimator va = new ValueAnimator();
-                va.setIntValues(getResources().getColor(R.color.red), getDrawingCacheBackgroundColor());
+                if (Build.VERSION.SDK_INT > Build.VERSION_CODES.KITKAT) {
+                    // only for kitkat and newer versions
+                    va.setIntValues(getResources().getColor(R.color.red),
+                            getDrawingCacheBackgroundColor());
+                    } else {
```

Comment:
The `}` should line up with the `if` it is connected to. Is it possible you're using tabs locally? All indentation should be done with spaces.

Reply:
Yes @lognaturel, I was using tabs..:(
I'll use spaces from now on.